assets/css/style.css

```diff
@@ -0,0 +1,7 @@
+header{
+font-family: 'Poppins', sans-serif;
+font-family: 'Quicksand', sans-serif;
+font-family: 'Satisfy', cursive;
```

Comment:
Just as a heads up, you will ONLY see the font-family of Satisfy for the header because it's the last one on the header.